pkg/tenant/members_test.go

```diff
@@ -51,6 +80,31 @@ func (suite *tenantTestSuite) TestTenantMemberCreate() {
 	require.Equal(req.Role, member.Role, "member role should match")
 }
 
+func (suite *tenantTestSuite) TestMemberList() {
+	require := suite.Require()
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+
+	defer cancel()
+
+	// Connect to mock trtl database.
+	trtl := db.GetMock()
+	defer trtl.Reset()
+
+	// Call the OnCursor method.
+	trtl.OnCursor = func(cr *pb.CursorRequest, t pb.Trtl_CursorServer) error {
+		return nil
+	}
+
+	req := &api.PageQuery{
+		PageSize:      2,
+		NextPageToken: "12",
+	}
```

Comment:
Just curious -- is this necessary for the test or just a rule of thumb for Ensign database calls? What happens if no pagination instructions are specified here (and same question for lines 30-33)?

Reply:
A `PageQuery` must be passed in for all of the list functions, but this will work if pagination instructions aren't included.